src/dayview.html

```diff
@@ -1,187 +1,181 @@
+                  </ng-template>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </init-position-scroll>
+        <init-position-scroll *ngIf="sliderIndex!==currentViewIndex" class="dayview-normal-event-container"
+          [initPosition]="initScrollPosition">
+          <table class="table table-bordered table-fixed dayview-normal-event-table">
+            <tbody>
+              <tr *ngFor="let tm of views[sliderIndex].rows; let i = index">
+                <td class="calendar-hour-column text-center">
+                  {{hourColumnLabels[i]}}
+                </td>
+                <td class="calendar-cell">
+                  <ng-template [ngTemplateOutlet]="dayviewInactiveNormalEventSectionTemplate"
+                    [ngTemplateOutletContext]="{tm:tm, hourParts: hourParts, eventTemplate:dayviewNormalEventTemplate}">
+                  </ng-template>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </init-position-scroll>
+      </ng-container>
+
+      <ng-container *ngIf="dayviewShowCategoryView">
+        <div *ngIf="sliderIndex===currentViewIndex" class="dayview-category-header">
+          <table class="table table-fixed dayview-category-header-table">
+            <thead>
+              <tr>
+                <th class="dayview-category-header-item calendar-hour-column"></th>
+                <th *ngFor="let category of dayviewCategorySource" class="dayview-category-header-item">
+                  <ng-template [ngTemplateOutlet]="dayviewCategoryItemTemplate" [ngTemplateOutletContext]="{category}">
+                  </ng-template>
+                </th>
+              </tr>
+            </thead>
+          </table>
+        </div>
+        <div *ngIf="sliderIndex!==currentViewIndex" class="dayview-category-header">
+          <table class="table table-fixed dayview-category-header-table">
+            <thead>
+              <tr>
+                <th class="dayview-category-header-item calendar-hour-column"></th>
+                <th *ngFor="let category of dayviewCategorySource" class="dayview-category-header-item">
+                  <ng-template [ngTemplateOutlet]="dayviewCategoryItemTemplate" [ngTemplateOutletContext]="{category}">
+                  </ng-template>
+                </th>
+              </tr>
+            </thead>
+          </table>
+        </div>
+
+        <div class="dayview-allday-table">
+          <div class="dayview-allday-label">{{ allDayLabel }}</div>
+          <div class="dayview-allday-content-wrapper scroll-content">
+            <table class="table table-fixed dayview-allday-category-table">
+              <tbody>
+                <td class="dayview-allday-category-cell" *ngFor="let category of dayviewCategorySource">
+                  <table class="table dayview-allday-content-table">
```

Comment:
I don't think it's necessary to nest a table inside dayview-allday-category-table, it seems just a one cell table.